client/my-sites/stats/activity-log-day/index.jsx

```diff
@@ -27,6 +32,10 @@ class ActivityLogDay extends Component {
 		disableRestore: false,
 		disableBackup: false,
 		isRewindActive: true,
+		rewindState: {
+			state: 'unavailable',
+			canAutoconfigure: false,
+		},
```

Comment:
hrm, this is a bit unsettling here. the selector should return the appropriate state whenever called, regardless of whether we have loaded it from the API yet. why are we setting a default value for it?

Reply:
This is a default for the worse scenario. Is this bad? Why? Because it might hide data fetch errors? Something else?